manual_refresh_token_provider.go

```diff
@@ -0,0 +1,33 @@
+package incognia
+
+import "errors"
+
+var ErrNoToken = errors.New("no token stored in memory")
```

Comment:
Suggestion with a new name for `ErrNoToken` and an error for when the token is expired.

```suggestion
var (
	ErrTokenNotFound = errors.New("token not found in memory")
	ErrTokenExpired  = errors.New("token is expired")
)
```